agency/usr/emiso_engine/cli/cli.cpp

```diff
@@ -0,0 +1,121 @@
+/*
```

Comment:
A detail, but could we call the app emiso-cli ? or emisocli ?  (just to avoid cli which is a bit fuzzy)

Reply:
The Cli class is part of the "emiso" name space. Is it not enough ?

Reply:
Yes, this is great, but at the end do we have a "cli" app ?

Reply:
The application name is `emiso_engine`. It offers 2 interfaces: `webserver` or `cli`. 

The `webserver` is the standard interfaces. The communication with Portainer is done through this interface. 
The `cli` is mainly used to test/debug the daemon. 

In current implementation, only one interface is exposed. The interface is selected by the options passed to the app.

Reply:
That's perect. Thanks for your explanation. It's OK with me.